nil BirthdayState defererence

In our dcrdex simnet harnesses, currently v2.0.1, I've seen a panic a couple of times at https://github.com/decred/dcrwallet/blob/f8cafb36df2764c677f0c2c951f9a797a5cfe147/wallet/udb/txmined.go#L352

Confirmed on v2.0.1.
@JoeGruffins?

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x41 pc=0x896bea]

goroutine 1 [running]:
decred.org/dcrwallet/v4/wallet/udb.SetBirthState({0x100c420?, 0xc000080470?}, 0x0)
        /home/buck/github/decred/dcrwallet/wallet/udb/txmined.go:352 +0x6a
decred.org/dcrwallet/v4/wallet.(*Wallet).SetBirthState.func1({0x100c420?, 0xc000080470?})
        /home/buck/github/decred/dcrwallet/wallet/rescan.go:501 +0x25
decred.org/dcrwallet/v4/wallet/walletdb.Update({0x1009250, 0xc0000b29b0}, {0x100a100, 0xc00027c240}, 0xc0002207f0)
        /home/buck/github/decred/dcrwallet/wallet/walletdb/interface.go:252 +0x190
decred.org/dcrwallet/v4/wallet.(*Wallet).SetBirthState(0xc000041000?, {0x1009250?, 0xc0000b29b0?}, 0xc000219101?)
        /home/buck/github/decred/dcrwallet/wallet/rescan.go:500 +0x4e
main.createWallet({0x1009250, 0xc0000b29b0}, 0xc000083c00)
        /home/buck/github/decred/dcrwallet/walletsetup.go:251 +0x756
main.loadConfig({0x1009250, 0xc0000b29b0})
        /home/buck/github/decred/dcrwallet/config.go:682 +0x1ec7
main.run({0x1009250?, 0xc0000b29b0})
        /home/buck/github/decred/dcrwallet/dcrwallet.go:82 +0x4c
main.main()
        /home/buck/github/decred/dcrwallet/dcrwallet.go:53 +0x4a
```

--- a/walletsetup.go
+++ b/walletsetup.go
@@ -169,7 +169,14 @@ func createWallet(ctx context.Context, cfg *config) error {
 		// Ask for a birthday if the wallet was created from seed. If
 		// the wallet is new the birthday is now. Add a day buffer to be safe.
 		if imported {
-			birthday, birthblock, err := prompt.Birthday(r)
+			var (
+				birthday   *time.Time
+				birthblock *uint32
+			)
+			birthday, birthblock, err = prompt.Birthday(r)
+			if errors.Is(err, io.EOF) {
+				err = nil
+			}
 			if err != nil {
 				return
 			}

